examples/pi4_mesa_build/README.md

```diff
@@ -0,0 +1,103 @@
+# Kompute Pi4 Mesa Build Example
+
+The Raspberry Pi 4 is an impressive little computer when you consider that the Broadcom GPU is able to run 2x 4K displays. This example intends to show how to get Vulkan Kompute up and running on a Raspberry Pi 4. This has huge potential for edge processing using the power of the Pi 4 GPU.
```

Comment:
This is great - the way this is worded could make a really good blog post - once we merge this if you are interested in publishing one I'd be keen on adding that to the docs as well

Reply:
Yeah I would be keen to publish a blog post once everything is working and merged. There are a number of potentially interesting applications such as edge detection which I would be keen to compare performance of on the Pi CPU versus GPU.

Reply:
That would be awesome